data/crac-file/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/remedial_action/range_action/PstRangeTest.java

```diff
@@ -63,19 +85,40 @@ public void setUp() throws Exception {
         Mockito.when(range2.getRangeDefinition()).thenReturn(RangeDefinition.STARTS_AT_ONE);
         Mockito.when(range2.getMin()).thenReturn(7.);
 
-        NetworkElement networkElement = Mockito.mock(NetworkElement.class);
+        networkElement = Mockito.mock(NetworkElement.class);
+        Mockito.when(networkElement.getId()).thenReturn(networkElementId);
 
+        // clarifier l'usage qu'on a de pstRange et pstRange1
```

Comment:
:gb: :us: :canada: :australia: => :ok:
:fr: :de: :sweden: :es: => :no_entry_sign:

Reply:
I deleted many useless statements in the setup method